views/About.module.css

```diff
@@ -167,3 +174,57 @@
 	grid-template-columns: 24px 1fr;
 	padding-bottom: 15px;
 }
+.iconswrap {
+	margin: 0;
+	padding: 0 0 130px 0;
+	background: linear-gradient(
+			360deg,
+			rgba(255, 69, 91, 0.07) 0%,
+			rgba(255, 144, 69, 0) 30%
+		),
+		url(/wierd.svg);
+	background-position: bottom;
+	background-repeat: no-repeat;
+}
+.iconblock {
+	border: 2px solid rgba(248, 248, 252, 0.1);
+	position: relative;
+	flex-direction: column;
+	align-items: center;
+	width: 262px;
+	height: 254px;
+	margin-left: 80px;
+}
+.iconblockodd {
+	margin: 15px;
```

Comment:
This is why your blocks are not evenly spaced.

Reply:
mine
<img width="422" alt="Screen Shot 2021-08-20 at 12 58 43 PM" src="https://user-images.githubusercontent.com/77123368/130287246-8ad30441-ba52-46e6-8e02-8453a1447f2c.png">